pegasus/test/test_hourofcode_helpers.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../src/env'
+require 'rack/test'
+require 'minitest/autorun'
+
+class HourOfCodeHelpersTest < Minitest::Test
+  include Rack::Test::Methods
+
+  def build_rack_mock_session
```

Comment:
it's invoked internally by [rack-test](https://github.com/brynary/rack-test) but after looking at the [code](https://github.com/brynary/rack-test/blob/master/lib/rack/test/methods.rb#L6-L20) again it looks like i was doing it slightly wrong, we should be defining an `app` instance method to return the app directly rather than overriding `build_rack_mock_session` directly. I'll update this